openedx/features/edly/utils.py

```diff
@@ -0,0 +1,14 @@
+import jwt
+from django.conf import settings
+
+def encode_edly_user_info_cookie(cookie_data):
+    """
+    Returns JWT encoded string of cookie data
```

Comment:
This should be a proper sentence with dot `.` at the end: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
done